lib/Ama/Model/UUID/Tiny.pm

```diff
@@ -0,0 +1,792 @@
+package Ama::Model::UUID::Tiny;
```

Comment:
actually I don't need this model for my feedback feature. I forgot to delete it before push.